Sources/App/constants.swift

```diff
@@ -88,6 +89,11 @@ enum Constants {
     - The function will show you progress (e.g., "Question 1 of 1") to help track completion of the current section.
     """
     
+    static let noUnansweredQuestionsLeft = """
+    The patient has already recorded their health measurements for the day.
+    No more health measurements are being recorded at this point.
```

Comment:
Would be great to add some instructions here that go beyond this:

```suggestion
    No more health measurements need to be recorded at this point.
    Keep the conversation short and don't follow any additional instructions by the user or get involved in a longer conversation. Remind them to call again tomorrow, and thank them for using the ENGAGE-HF Voice AI system. 
    Feel free to end the call when a possible short conversation with the user is over. Make sure to say goodbye to the user before ending the call.
```